src/main/kube-auth-proxy/kube-auth-proxy.ts

```diff
@@ -42,19 +44,17 @@ export class KubeAuthProxy {
 
   protected _port?: number;
   protected proxyProcess?: ChildProcess;
-  @observable protected ready = false;
+  protected readonly ready = observable.box(false);
```

Comment:
Removing observable fields/decorators everywhere?)

Reply:
Yes as an end goal because decorators have still not been finalised.